packages/wallets/src/features/cashier/modules/WithdrawalCrypto/provider/WithdrawalCryptoProvider.tsx

```diff
@@ -161,9 +171,9 @@ const WithdrawalCryptoProvider: React.FC<React.PropsWithChildren<TWithdrawalCryp
         getConvertedFiatAmount,
         getCurrencyConfig: getConfig,
         isClientVerified: getClientVerificationStatus(),
+        isCryptoConfigLoading,
         isTokenValidationLoading,
```

Comment:
@lubega-deriv could we return isLoading: isCryptoConfigLoading || isTokenValidationLoading from provider and not add it into [WithdrawalCrypto.tsx](https://github.com/binary-com/deriv-app/pull/15293/files#diff-499eb17ee4e16473dcbaf9f73017fe73263e276715e9389027f1fbe65df5794a) line 38?